src/main/java/org/prgrms/kdt/controller/VoucherController.java

```diff
@@ -17,12 +17,12 @@ public VoucherController(VoucherService voucherService) {
         this.voucherService = voucherService;
     }
 
-    public boolean create(CreateVoucherRequest createVoucherRequest) {
+    public boolean createVoucher(CreateVoucherRequest createVoucherRequest) {
         CreateVoucherDto createVoucherDto = new CreateVoucherDto(createVoucherRequest.getVoucherType(), createVoucherRequest.getDiscountAmount());
         return voucherService.createVoucher(createVoucherDto);
     }
 
-    public List<VoucherResponse> list() {
+    public List<VoucherResponse> getAllVouchers() {
         List<Voucher> vouchers = voucherService.getAllVouchers();
         return vouchers.stream().map(VoucherResponse::new).toList();
```

Comment:
저는 stream의 경우 짧더라도 다음과 같이 다음 라인으로 보내 어떤 순서로 처리되는지 보기 쉽게 하는 편입니다. 이에대해 어떻게 생각하시나요?!
```suggestion
        return vouchers.stream()
        .map(VoucherResponse::new)
        .toList();
```

Reply:
보기 좋은 것 같네요! 반영하겠습니다